examples/pom.xml

```diff
@@ -173,6 +173,7 @@
                         <outputDirectory>${project.build.directory}/checkstyle</outputDirectory>
                         <outputFile>${project.build.directory}/checkstyle/checkstyle-result.xml</outputFile>
                         <configLocation>${basedir}/../etc/config/checkstyle.xml</configLocation>
+                        <failsOnError>true</failsOnError>
```

Comment:
-1 for failing the build, as this will stand in the way of quickly fixing things and correcting code style later. Please open a **separate PR** for this, as I requested from you, so we can *quickly* merge the actual checkstyle improvements without waiting for all committers to accept this rule change. Having said that, from now on, this PR fails the criteria for fast-track just due to your personal likes. :-(

Reply:
Moved this from #1026 to #1027 (cf. https://github.com/eclipse-ee4j/jaxrs-api/issues/978#issuecomment-932780285)

Reply:
Thanks! So we can merge *this* one quickly, while committers have the chance to discuss the rule change separately.